src/containers/FormikForm/formikDraftHooks.js

```diff
@@ -32,35 +33,63 @@ export function useFetchArticleData(articleId, locale) {
     if (articleId) {
       setLoading(true);
       const article = await draftApi.fetchDraft(articleId, locale);
+
+      const convertedConcepts = await fetchElementList(article.conceptIds);
       const taxonomy = await fetchTaxonomy(articleId, locale);
       setArticle(
-        transformArticleFromApiVersion({ taxonomy, ...article }, locale),
+        transformArticleFromApiVersion(
+          { taxonomy, ...article },
+          locale,
+          convertedConcepts,
+        ),
       );
       setLoading(false);
     }
   };
 
   const updateArticle = async updatedArticle => {
-    const savedArticle = await draftApi.updateDraft(updatedArticle);
+    const conceptIds = updatedArticle.conceptIds.map(concept => concept.id);
+    const savedArticle = await draftApi.updateDraft({
+      ...updatedArticle,
+      conceptIds: conceptIds,
```

Comment:
Trur du kan utelate navnet når const heiter det samme.
```suggestion
      conceptIds,
```